src/app/app.component.ts

```diff
@@ -50,6 +50,11 @@ export class AppComponent implements OnInit {
   private Config = this.config.getConfig();
   public title = this.Config.title || "Rancher's Hobby Farm";
   private logo = this.Config.logo || '/assets/default/logo.svg';
+  public aboutTitle = this.Config.about?.title || 'About HobbyFarm';
+  public aboutBody =
+    this.Config.about?.body ||
+    'HobbyFarm was lovingly crafted by @ebauman and @Oats87 from @RancherLabs';
```

Comment:
Thanks for the review @ebauman! Do you want me to display  the about body as a text or would it be nice to allow dynamic html?

I could change the default aboutBody value to:
```suggestion
  private hyperlink = '<a href="https://github.com/hobbyfarm/hobbyfarm" target="_blank">https://github.com/hobbyfarm/hobbyfarm</a>';
  public aboutBody =
    this.Config.about?.body ||
    `HobbyFarm is lovingly crafted by the HobbyFarm team at ${this.hyperlink}`;
```

And it would look like this:
![about_hobbyfarm](https://user-images.githubusercontent.com/38468835/161992864-54388921-82c4-4c8f-8656-04e01e2c7f3b.png)

Otherwise you would not be able to click on the link. The question that comes with this is whether we want to allow admins to mount dynamic HTML into the about section or not.

Reply:
I would say don't allow dynamic injection - sounds like a lot of pain for a relatively small portion of functionality. Let's remove the link from the text and leave that as "HobbyFarm is lovingly crafted by the HobbyFarm team".

Then one of the example buttons can be a link to the HF project.